src/webview_provider/docsEditPanel.ts

```diff
@@ -249,6 +253,74 @@ export class DocsEditViewPanel implements WebviewViewProvider {
     await this.resolveWebviewView(this.panel!, this.context!, this.token!);
   };
 
+  private getTestDataByColumn(message: any, columnName: string) {
+    const tests = message.tests as undefined | TestMetaData[];
+
+    if (!tests?.length) {
+      this.terminal.debug("tests", "No test data passed");
```

Comment:
"tests" may look a bit generic if seeing that in the logs. Can you put the class name instead?

Reply:
fixed